ibis/expr/types/relations.py

```diff
@@ -2822,7 +2822,7 @@ def info(self) -> Table:
                 nulls=lambda t: t.isna.sum(),
                 non_nulls=lambda t: (1 - t.isna).sum(),
                 null_frac=lambda t: t.isna.mean(),
-                pos=lit(pos),
+                pos=lit(pos).cast(dt.int16),
```

Comment:
```suggestion
                pos=lit(pos, type=dt.int16),
```

instead of the literal creation and then the cast, you can specify the type of the literal explicitly

Reply:
I would just specify `int32` to avoid having to do this again when someone requests > 2**15-1 columns. Hopefully we never get there.

Reply:
true, perhaps int16 is fine for now then